src/Components/Body/Body.tsx

```diff
@@ -8,30 +8,37 @@ interface bodyProps {
 }
 interface bodyObj {
     Channels: string[],
-    DM: string[],
-    Application: string[],
+    DirectMessages: string[],
+    Apps: string[],
```

Comment:
variable names should start with a small case letter - DirectMessages -> directMessages, Apps -> apps
interface name should start with a capital letter
check for each such occurrences